packages/semantic-ui/src/URLWidget/URLWidget.js

```diff
@@ -22,7 +22,14 @@ function URLWidget(props) {
     uiSchema,
     formContext,
   } = props;
-  const semanticProps = getSemanticProps({ formContext, options });
+  const semanticProps = getSemanticProps(
+    { formContext, options,
+      uiSchema,
+      defaultSchemaProps: {
+        fluid: true,
+        inverted: false,
```

Comment:
Consider moving `options` to a separate line to be consistent with most of the rest of the code. Also, consider not passing `defaultSchemaProps` since it has the same props set as the default value when it is not provided.